eng/pipelines/npm-tasks.yml

```diff
@@ -0,0 +1,67 @@
+trigger: none
+
+# This pipeline helps to run NPM admin tasks like remove or add tag to a released package version or deprecate a pacakge version
+# Following variables should be set at queue time to run this pipeline
+# variable name: TaskType
+# valid Options: 'AddTag', 'RemoveTag', 'DeprecatePackage'
+
+# variable name: PackageName
+# e.g. values: '@azure/storage-blob', '@azure/core-http'
+
+# variable name: 'PkgVersion'
+# e.g. values '1.0.0', '2.1.0.beta.1'
+
+# variable name: 'TagName'
+# e.g values: 'latest', 'next'
+
+parameters:
+  - name: TaskType
+    displayName: Task Type
+    type: string
+    default: AddTag
+    values:
+    - AddTag
+    - RemoveTag
+    - DeprecatePackage
+
+  - name: PackageName
+    displayName: Package Name (e.g. @azure/template)
+    type: string
+    default: ''
+
+  - name: PkgVersion
+    displayName: Package Version (e.g. 1.1.0)
+    type: string
+    default: ''
+
+  - name: TagName
+    displayName: Tag (e.g. latest, next)
+    type: string
+    default: ''
+
+  - name: Reason
+    displayName: Reason
```

Comment:
Might be worth adding "Reason for deprecation" to make it clear that is only for deprecation.